app/admin/imports/template.hbs

```diff
@@ -0,0 +1,50 @@
+<div class="admin container">
+  <div class="page-header">
+    <h1>Accounts</h1>
+  </div>
+  <div class="text-center">
+    <div class="clearfix">
+      <div class="form-group">
+      <img src="images/icon-date.svg" width="34" class="date-icon item-inblock margin-r5" />
+      <div class="date-filter item-inblock">
+      {{bootstrap-datepicker
+          class="calendar"
+          orientation="top"
+          autoclose=true
+          value=atDate
+          endDate=endsAt
+      }}
+    </div>
+  </div>
+</div>
+</div>
+  {{#if reports.length}}
+  <table class="table table-bordered">
+    <thead>
+      <tr>
+        <th>
+          Name
+        </th>
+        <th>
+          Amount
```

Comment:
Should be `Total Jobs` as requested by Ryan: https://github.com/bcjobs/analytics/issues/179#issue-147481409